core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamTest.scala

```diff
@@ -21,6 +21,11 @@ import org.scalatest.{Matchers, FunSuite}
 class SimpleDateParamTest extends FunSuite with Matchers {
 
   test("date parsing") {
+    import java.util.{Locale, TimeZone}
```

Comment:
If so, then the whole test framework should set a default timezone globally. However, it's really not good to rely on this.

There seems to be a bug as this logic really can't depend on the current machine's timezone. "2015-02-20T23:21:17.190GMT" unambiguously names a point in time. So I don't htink we can commit this as a fix but rather root out the bug.